testGameSearch/Game.swift

```diff
@@ -8,13 +8,14 @@
 
 import Foundation
 
-struct Game: Codable, Identifiable {
+struct Game: Codable, Identifiable, Hashable {
```

Comment:
Need to make `Game` conform to `Hashable` so that it can be used in the `ForEach` in the view